src/fairplay.js

```diff
@@ -1,3 +1,6 @@
+/**
+ * @see https://www.w3.org/TR/2014/WD-encrypted-media-20140218
```

Comment:
I'm honestly not sure. Getting any information on the spec / version implemented for Safari and IE11 is extremely difficult. I went based on the fact that it looks like that version. Looking a little deeper, it looks like maybe it was an older version, since in the 2014 draft the element attribute is `mediaKeys`, while in earlier versions it's just `keys`, and the code that's already here uses `webkitKeys`, which makes me think it's an earlier one.

I'll do a bit closer of a comparison and see if I can pin down an exact version.... unless you know of somewhere that explicitly states the version/API.

Reply:
Turns out it looks more like https://www.w3.org/TR/2013/WD-encrypted-media-20131022, though obviously it's impossible to know for sure.